packages/sdk/client/src/echo/space-proxy.ts

```diff
@@ -131,6 +131,7 @@ export class SpaceProxy implements Space {
     if (this._properties) {
       return this._properties;
     } else {
+      log('Using cached properties');
```

Comment:
nit: i'm not strongly defending this, but the logging pattern that has emerged is all lower case with an optional object. vs. "Errors which are english like sentences with a period."